Planner/src/Planner.java

```diff
@@ -322,28 +378,38 @@ public void actionPerformed(ActionEvent arg0) {
 			    	@Override
 					public void actionPerformed(ActionEvent arg0) {
 			    		int i = usersList.getIndexOf(editUser);
-			    		boolean result = UserDAO.deleteUser(editUser); 
-			    		if (result == true) {
-			    			usersList.removeElementAt(i);
-			    		}
+			    		try {
+			    			boolean result = UserDAO.deleteUser(editUser); 
+			    			if (result == true) {
+			    				usersList.removeElementAt(i);
+			    			}
+			    	
 			    		editUserFrame.dispose();
+			    		} catch (SQLException e) {
+			    			JFrame frame = new JFrame();
+			    			JOptionPane.showMessageDialog(frame,
+			    				"You can't delete the user, who is assigned to the task.",
```

Comment:
Maybe we can change this message to: "This user can not be deleted as it has one more tasks assigned to him."